main.css

```diff
@@ -338,10 +341,40 @@ a {
   font-size: 1.2em;
 }
 
+
 /*********************************/
 
 @media only screen and (max-width: 991px) {
-    .navbar-nav .links{
-        margin: 0.25rem auto;
+
+    .title{
+      background-color: pink;
```

Comment:
In regards to colours, it could be nice to have some consistency. E.g. whether we use words, hex or RGB. My personal preference is hex.

Reply:
On this point, why is the background colour changed to pink?
![image](https://user-images.githubusercontent.com/54030147/158056930-9fc26848-5208-40d0-91bb-c5f85f1654c8.png)

Reply:
agree with this as well we should try to keep the color consistently , not sure why its pink either.

Reply:
I was just checking whether the responsiveness is working or not that's why I added that pink color but forgot to remove it before making PR.